### PR TITLE
Add test parameters to reduce debug.log output

### PR DESCRIPTION
**Pull request by yanglei-rh**
1.debug.log is too big to load

ID: 1850346
Signed-off-by: Lei Yang <leiyang@redhat.com>

### CODE REVIEW

**Comment by yanglei-rh** [comment]
@PaulYuuu Please help me review it,thanks.

**Comment by PaulYuuu** [comment]
cc @CongLi

**Comment by PaulYuuu** [comment]
Would you also handle the tcpdump output?

File: qemu/tests/pxe_query_cpus.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 @error_context.context_aware
 def _capture_tftp(test, vm, timeout):
     error_context.context("Snoop packet in the tap device", logging.info)
-    output = aexpect.run_fg("tcpdump -nli %s" % vm.get_ifname(),
+    output = aexpect.run_fg("tcpdump -nli %s |grep -i tftp" % vm.get_ifname(),
```
Comment (PaulYuuu) [inline qemu/tests/pxe_query_cpus.py:18]: FYI: https://github.com/autotest/tp-qemu/pull/1931

And if you want to capture some messages, you can filter out `not arp and not icmp` or other.

Comment (yanglei-rh) [reply qemu/tests/pxe_query_cpus.py]: I have updated it, please help me review it again.Thanks.

**Comment by PaulYuuu** [comment]
LGTM.
Can you provide the test result?

**Comment by PaulYuuu** [comment]
The command format: https://stackoverflow.com/questions/2187932/monitoring-multiple-ports-in-tcpdump

**Comment by yanglei-rh** [comment]
Test Pass:
(1/2) Host_RHEL.m8.u3.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.pxe_boot.q35: PASS (23.55 s)
 (2/2) Host_RHEL.m8.u3.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.pxe_boot.with_query_cpus.q35: PASS (518.55 s)

**Comment by yanglei-rh** [comment]
@CongLi  Please help me review it,thanks.

**Comment by CongLi** [comment]
ack.

**Pull request merged**